flink-core/src/test/java/org/apache/flink/util/StringUtilsTest.java

```diff
@@ -39,21 +40,26 @@ public void testControlCharacters() {
 	}
 
 	@Test
-	public void testArrayOfBytesArray() {
-		byte[][] expectedArray = new byte[][]{
-			{1, -97, 49, 74 },
-			{2, -92, 48, 73 }
-		};
+	public void testArrayAwareToString() {
```

Comment:
nit: I very much prefer to have a single case in a single `Test` method. It makes it much easier to track back the failing case. Moreover a subsequent assertions/cases are run even if an earlier one fails.

Reply:
Currently, every test method tests one util method. I would like to keep it as it is, if that's ok. I think it is fine to fail early if the first assertion fails.